### PR TITLE
Add rayo.js to the tests.

### PR DESCRIPTION
**Pull request by aichholzer**
@waghanza -As mentioned [here](https://github.com/tbrand/which_is_the_fastest/issues/237)
Thank you.

### CODE REVIEW

File: node/rayo/package.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "author": {
+    "name": "Stefan Aichholzer",
+    "email": "theaichholzer@gmail.com",
+    "url": "https://github.com/aichholzer"
+  },
+  "dependencies": {
+    "rayo": "^1.0.4"
```
Comment (waghanza) [inline node/rayo/package.json]: I prefer using `~` instead of `^`

Comment (aichholzer) [reply node/rayo/package.json]: Noted, thank you.

**Comment by aichholzer** [comment]
@waghanza -How are you running the other Nodejs frameworks (express, fastify, polka)? -None of them implements a `HEAD` endpoint (which is what `curl -vIL` is doing) and they all return `404` on that request.

**Comment by waghanza** [comment]
Yes. I'm confusing myself. Juste wating to add some tests

**Comment by aichholzer** [comment]
@waghanza -Let me know if there is anything I can help you with.

**Comment by waghanza** [comment]
curl -vIL could be used to display http headers

**Comment by aichholzer** [comment]
@waghanza -Yes, but the server needs to implement a `HEAD` endpoint, for instance:
```js
const rayo = require('rayo');
rayo({ port: 3000 })
  .head('/', (req, res) => res.end(''))
  .start();
```

`curl -vIL` basically makes a `HEAD` request to the server.

**Pull request merged**